sw/host/opentitanlib/src/transport/verilator/transport.rs

```diff
@@ -84,8 +84,8 @@ impl Drop for Verilator {
 }
 
 impl Transport for Verilator {
-    fn capabilities(&self) -> Capabilities {
-        Capabilities::new(Capability::UART)
+    fn capabilities(&self) -> Result<Capabilities, TransportError> {
```

Comment:
Isn't `TransportError` [already included](https://github.com/lowRISC/opentitan/blob/8c96f952159b03eba5cebfa2af3f3606d23909ad/sw/host/opentitanlib/src/transport/errors.rs#L71)? Or am I missing something?

Reply:
This file has `use anyhow::Result` at the top, which means that the default is anyhow::Error, if a particular error type is not included.
With this change, there are three instances of Result in this file that uses TransportError, and two that uses anyhow::Error.  I could replate `use anyhow::Result` with `use crate::transport::Result`, and then edit the two places that do not use TransportError to explicitly put `anyhow::` prefix on their `Result` type.  I have done so in other files where the of `TransportError` to `anyhow::Error` is higher.

Reply:
Ah I see. That sounds like it would make things more clear / consistent.

Reply:
Done.